presto-connector-tests/src/main/java/com/facebook/presto/connector/unittest/BaseMetadataTest.java

```diff
@@ -77,7 +82,7 @@ default void testEmptyMetadata()
         withMetadata(
                 ImmutableList.of(
                         metadata -> assertEquals(metadata.listSchemaNames(session), systemSchemas()),
-                        metadata -> assertEquals(metadata.listTables(session, null), ImmutableList.of())));
+                        metadata -> assertEquals(metadata.listTables(session, null), systemTables())));
```

Comment:
this belongs to one of the previous commit

Reply:
The code in the previous commits evolved with the addition of this one. The change is fine here.